addons/project/static/src/js/project_form.js

```diff
@@ -17,8 +17,7 @@ const ProjectFormController = FormController.extend({
     _onDomUpdated() {
         const $editable = this.$el.find('.note-editable');
         if ($editable.length) {
-            const minHeight = window.innerHeight - $editable.offset().top - 42;
-            $editable.css('min-height', minHeight + 'px');
+            $editable.css('min-height', '120px');
```

Comment:
I guess the fix should be to change the selector to only select the description field.